### PR TITLE
Fix android sdkmanager installation path

### PR DESCRIPTION
**Pull request by vsafonkin**
Issue: [#60](https://github.com/actions/virtual-environments/issues/60)

It fixes installation process to install sdkmanager to the correct directory. It changes directory for sdk manager from `/usr/local/lib/android` to `/usr/local/lib/android/sdk` ($ANDROID_HOME).

Builds:
[Ubuntu 1604](https://dev.azure.com/github/virtual-environments/_build/results?buildId=62040)
[Ubuntu 1804](https://dev.azure.com/github/virtual-environments/_build/results?buildId=62041)

### CODE REVIEW

File: images/linux/scripts/installers/1804/android.sh [hunk 1]
```diff
@@ -19,11 +19,11 @@ echo "ANDROID_HOME=${ANDROID_SDK_ROOT}" | tee -a /etc/environment
 # Download the latest command line tools so that we can accept all of the licenses.
 # See https://developer.android.com/studio/#command-tools
 wget -O android-sdk.zip https://dl.google.com/android/repository/sdk-tools-linux-4333796.zip
-unzip android-sdk.zip -d ${ANDROID_ROOT}
+unzip android-sdk.zip -d ${ANDROID_SDK_ROOT}
 rm -f android-sdk.zip
 
 # Install the following SDKs and build tools, passing in "y" to accept licenses.
-echo "y" | ${ANDROID_ROOT}/tools/bin/sdkmanager --sdk_root=${ANDROID_SDK_ROOT} \
+echo "y" | ${ANDROID_SDK_ROOT}/tools/bin/sdkmanager --sdk_root=${ANDROID_SDK_ROOT} \
```
Comment (alepauly) [inline images/linux/scripts/installers/1804/android.sh]: are there any checks/verifications we can add to make sure this doesn't regress?

Comment (vsafonkin) [reply images/linux/scripts/installers/1804/android.sh]: @alepauly , I've added check for installation sdk manager in `$ANDROID_SDK_ROOT` (the same `$ANDROID_HOME`).

**Pull request merged**